### DISCUSSION TITLE
Audio streaming support

### DISCUSSION THREAD

**Discussion by Oubi256** [alnitak/flutter_soloud#77] [Q&A]
At the moment the library can only work with downloaded files, is it possible to make it work with audio streaming (internet radio, etc.)? Maybe some kind of workaround?

**Comment by alnitak** [comment]
Hi @Oubi256 ,

currently, it is not yet possible for example to feed the audio callback with a stream of data.
It would be nice to have RTSP capability; I don't see a workaround now.
I think the best way could be done by creating a [new audio source on C side](https://solhsa.com/soloud/newsoundsources.html).
